website_chd_product_configurator/controllers/controllers.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+from openerp import http
+from openerp.http import request
+from openerp.osv import orm
+import json
+
+
+class Chd_website(http.Controller):
+
+    @http.route('/chd_init/', auth='public', website=True)
+    def start(self, selected_id=False, type=False):
+        partner = self.get_current_partner()
+        if partner is False:
+            return request.render(
+                'website_chd_product_configurator.no_partner', {})
+        Conf_products = request.env['product.template']
+        accessories = request.env['product.product']
+        chd_price_components_at = request.env[
+            'price.component.attribute.template']
+        chd_types = request.env['product.type']
+        chd_finishing = request.env['product.finishing']
+        # when posting a selected product to configure
+        if request.httprequest.method == 'POST' and selected_id:
```

Comment:
And you probably want to be sure selected_id is something we can make an integer from (`string.is_digit()`), otherwise the orm can give some funny results